### DISCUSSION TITLE
Maintaining Gdevelop own fork of piskel

### DISCUSSION THREAD

**Discussion by Midhil457** [4ian/GDevelop#3670] [Enhancement ideas (and their technical discussions)]
It seems that piskel is slowly becoming stale. Their last push was on Jul 2021, https://github.com/piskelapp/piskel
And the number of bugs being reported about piskel is increasing day by day

I suggest maintaining a seperate fork of piskel. It is not necessary that it should be actively updated. But, it will let GDevelop contributors fix and change to fit GDevelop.

I don't think it will be a big burden for the team. It can just exist as another repo and contributors who wish to contribute to GDevelop's piskel can open issues and PRs there. Currently, the community and contributors don't have control over piskel which imo is a bad thing cuz piskel is used a lot and is very essential. If you watch some GDevelop review (like from gfs), they show piskel as a feature and I think it is important to see it as a essential part of GDevelop and try maintaining it on GDevelop's on terms
There was some unofficial talk and suggestion about replacing piskel already, Why not change piskel to fit GDevelop's needs?
Yes, jfxr and yarn are also embedded into Gdevelop like piskel. But, piskel is used way more than them and more important.

:)

**Comment by AlexandreSi** [comment]
I think you got a point here, especially because it was kind of already the case, we use a fork of @blurymind at the moment.

If I list the issues I see with Piskel, this is what I got:
- Tooltip broken #3615 
- Loading problems #3200
- Other display problem #1751
- UX problem with Piskel own saving system #4245 

Are there other issues that I'm missing?

**Comment by Midhil457** [comment]
One other issue I have come across is that piskel doesn't do a good job at transparency
- And ones an image is opened and saved using piskel, the quality of the image (anti-aliasing) gets removed
